_data/collection-index.yml

```
@@ -923,3 +923,8 @@
   contact: https://github.com/lx-0/devcontainer-templates/issues
   repository: https://github.com/lx-0/devcontainer-templates
   ociReference: ghcr.io/lx-0/devcontainer-templates
+- name: "Hauke's Features for Development Containers"
```

```suggestion
- name: Hauke's Features for Development Containers
```

Can we remove the quotes? The crawler action would fail to parse this, thanks!

---

Did that, though the double quotes should be standard YAML - what YAML parser is the crawler using?